wasm/module_test.go

```diff
@@ -173,3 +174,93 @@ func TestDuplicateExportError_NoStackOverflow(t *testing.T) {
 	err := wasm.DuplicateExportError("h")
 	_ = err.Error()
 }
+
+func TestGetFuntionSig(t *testing.T) {
+	f, err := os.Open("testdata/spec/sigtest.wasm")
+	if err != nil {
+		t.Fatalf("%v", err)
+	}
+	defer f.Close()
+	m, err := wasm.ReadModule(f, nil)
+	if err != nil {
+		t.Fatalf("error reading module %v", err)
+	}
+
+	// check first sig
+	fsig, err := m.GetFunctionSig(0)
```

Comment:
these `m.GetFunctionSig(0,1,2,3)` could be rewritten as a table-driven test.
but ok, we can leave it for another PR :)